src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -104,15 +102,25 @@ def format_open_pr_comment_subtitle(title_length, subtitle):
 
 
 # for a single file, create a table
-def format_issue_table(diff_filename: str, issues: List[PullRequestIssue], toggle=False) -> str:
+def format_issue_table(
+    diff_filename: str, issues: List[PullRequestIssue], patch_parsers: Dict[str, Any], toggle: bool
+) -> str:
+    language_parser = patch_parsers.get(diff_filename.split(".")[-1], None)
+
+    if not language_parser:
+        return ""
+
+    issue_row_template = language_parser.issue_row_template
+
     issue_rows = "\n".join(
         [
-            OPEN_PR_ISSUE_ROW_TEMPLATE.format(
+            issue_row_template.format(
                 title=issue.title,
                 subtitle=format_open_pr_comment_subtitle(len(issue.title), issue.subtitle),
                 url=format_comment_url(issue.url, GITHUB_OPEN_PR_BOT_REFERRER),
                 event_count=small_count(issue.event_count),
                 function_name=issue.function_name,
+                affected_users=small_count(issue.affected_users),
```

Comment:
the python issue row template will ignore the `affected_users` argument because it doesn't use it